docs/topics/proc_running-the-containerless-mta-cli.adoc

```diff
@@ -31,17 +31,23 @@ NOTE: If you do not set `JVM_MAX_MEM`, the analysis might hang.
 .Procedure
 
 . Move the installed {ProductShortName} CLI `.zip` file to your `$PATH`:
+.. For example, using `sudo`:
```

Comment:
I think using 'sudo' is redundant since you write it in the command below

Reply:
so, for **Linux**:

- Unzip dependency zip to home folder / .kantra
`unzip $HOME/mta-cli.<os>.<arch>.zip -d $HOME/.kantra`

- Put product binary to `/usr/bin`
`sudo mv ~/.kantra/mta-cli /usr/bin/`

OR

- Add `~/.kantra` path to $PATH variable
`export PATH=$HOME/.kantra:$PATH`
Advantage of second option is that it doesn't require root permissions